tests/test_main.py

```diff
@@ -19,21 +20,36 @@
 
 
 @pytest.mark.parametrize(
-    ('cli_args', 'build_args'),
+    ('cli_args', 'init_args', 'build_args'),
     [
-        ([], [cwd, out, ['sdist', 'wheel'], False]),
-        (['-s'], [cwd, out, ['sdist'], False]),
-        (['-w'], [cwd, out, ['wheel'], False]),
-        (['source'], ['source', out, ['sdist', 'wheel'], False]),
-        (['-o', 'out'], [cwd, 'out', ['sdist', 'wheel'], False]),
-        (['-x'], [cwd, out, ['sdist', 'wheel'], True]),
+        (['-n'],              [cwd],      [out, ['sdist', 'wheel'], False]),
+        (['-n', '-s'],        [cwd],      [out, ['sdist'], False]),
+        (['-n', '-w'],        [cwd],      [out, ['wheel'], False]),
+        (['-n', 'source'],    ['source'], [out, ['sdist', 'wheel'], False]),
+        (['-n', '-o', 'out'], [cwd],      ['out', ['sdist', 'wheel'], False]),
+        (['-n', '-x'],        [cwd],      [out, ['sdist', 'wheel'], True]),
     ]
 )
-def test_parse_args(mocker, cli_args, build_args):
-    mocker.patch('build.__main__.build')
+def test_parse_args(mocker, cli_args, init_args, build_args):
+    mocker.patch('build.__main__.MainBuilder.__init__', return_value=None)
+    mocker.patch('build.__main__.MainBuilder.get_dependencies')
+    mocker.patch('build.__main__.MainBuilder.build_distributions')
 
     build.__main__.main(cli_args)
-    build.__main__.build.assert_called_with(*build_args)
+    build.__main__.MainBuilder.build_distributions.assert_called_with(*build_args)
+    build.__main__.MainBuilder.__init__.assert_called_with(*init_args)
+
+
+@pytest.mark.skipif(sys.version_info[:2] == (3, 5), reason='bug in mock')
```

Comment:
generally, I tend to put here a link to the bug report 🤔 because other people will not know what you're talking about, without 🤔

Reply:
I was too lazy to do that :P. Anyway, it is actually not a bug, I was too quick to judge the backtrace. It is fixed now.